components/group-mgt/org.wso2.carbon.device.mgt.group.core/src/main/java/org/wso2/carbon/device/mgt/group/core/dao/GroupManagementDAOUtil.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Fixed in https://github.com/wso2/carbon-device-mgt/commit/e1cd4b1907f419f974545fc214c58cd9f247f2b2